Userland/Libraries/LibGfx/QOICommon.h

```diff
@@ -33,4 +38,9 @@ struct [[gnu::packed]] QOIHeader {
     u8 colorspace;
 };
 
+ALWAYS_INLINE static constexpr bool qoi_is_valid_run(u8 run)
```

Comment:
I'd say all this stuff has to go in `namespace Gfx::QOI`, `qoi_is_valid_run` is not a very C++ name.